src/apps/VirtualApp/VirtualChat.js

```diff
@@ -13,6 +19,8 @@ class VirtualChat extends Component {
     support_msgs: [],
     room_chat: true,
     from: null,
+    // TODO: Share one delivery manager for all required components?
+    gdm: new GuaranteeDeliveryManager(MAX_DELAY, RETRY_DELAY, INTERVALS_DELAY),
```

Comment:
Empty constructor should use sane defaults. 
No need to pass these around

Reply:
Fixed by default values.